### PR TITLE
Add UI

### PR DESCRIPTION
**Pull request by lilianaweimer**
Adds animations & logo!

This is what Copilot has to say about it:

This pull request primarily updates the `coinflip Watch App` to improve the user interface and add animations. The most significant changes include adding a new logo to the app icon set, introducing a new state variable to manage the display of text, and overhauling the body of the `ContentView` to include animations and improve the layout.

User Interface Enhancements:
* [`coinflip Watch App/Assets.xcassets/AppIcon.appiconset/Contents.json`](diffhunk://#diff-fc6f99e0ee37ca8ef8281b71cd5642df4f0ae08554d098889fa42e549f616055R4): Added a new logo, `CoinflipLogo.png`, to the app icon set.

State Management:
* [`coinflip Watch App/ContentView.swift`](diffhunk://#diff-98b34767917411d779f5d9bce4204bb59059ff7dfcb20b5aa71cd5009a0e3214R6-R95): Introduced a new state variable, `isShowingText`, to manage the display of text in the view.

Layout and Animation Improvements:
* [`coinflip Watch App/ContentView.swift`](diffhunk://#diff-98b34767917411d779f5d9bce4204bb59059ff7dfcb20b5aa71cd5009a0e3214R6-R95): Replaced the previous `body` of the `ContentView` with a new layout that includes a `ZStack` and `VStack` to improve the layout of elements. Also, introduced a new `flipAnimation` and applied it to a `Circle` view to create a flip effect when the coin is flipped. Furthermore, accessibility identifiers were added to several elements to improve accessibility.

### CODE REVIEW

File: coinflip Watch App/ContentView.swift [hunk 1]
```diff
@@ -3,37 +3,96 @@ import SwiftUI
 struct ContentView: View {
     @State private var coin = Coin()
     @StateObject private var sessionStats = CoinflipStats()
+    @State private var isShowingText = true
```
Comment (lilianaweimer) [inline coinflip Watch App/ContentView.swift:6]: this could have a better name tbh

File: coinflip Watch App/ContentView.swift [hunk 2]
```diff
@@ -3,37 +3,96 @@ import SwiftUI
...
+                        
+                        if (sessionStats.totalCount > 1) {
+                            VStack {
+                                Image(systemName: "cat")
+                                    .accessibilityIdentifier("TotalTailsCountIcon")
+                                Text(sessionStats.tailsCount.description)
+                                    .accessibilityIdentifier("TotalTailsCount")
+                            }
+                        }
+
```
Comment (CaptnBigShot) [inline coinflip Watch App/ContentView.swift:60]: ?

File: coinflip Watch App/ContentView.swift [hunk 3]
```diff
@@ -3,37 +3,96 @@ import SwiftUI
...
+                .stroke(Color.white, lineWidth:2)
+                .scaleEffect(isShowingText ? 1 : 2.5)
+                .animation(
+                    flipAnimation,
+                    value: coin.wasFlipped()
+                )
+                .blur(radius: (1) * 20)
+                .frame(width: .infinity, height: .infinity)
+                .padding(.top, -20)
+
```
Comment (CaptnBigShot) [inline coinflip Watch App/ContentView.swift:30]: ?

File: coinflip Watch App/ContentView.swift (hunk 2 above)
Comment (lilianaweimer) [reply coinflip Watch App/ContentView.swift:60]: that's xcode's fault

**Review by CaptnBigShot — APPROVED** [review]
[lgtm.txt](https://github.com/CaptnBigShot/simple-coinflip/files/15049383/lgtm.txt)

**Pull request merged**